src/NUnitConsole/nunit3-console/OptionsUtils/OptionParser.cs

```diff
@@ -0,0 +1,132 @@
+﻿// ***********************************************************************
```

Comment:
All this code is just moved out of the original two CommandLineOptions classes, to try and limit the file size.